appveyor.yml

```diff
@@ -14,3 +14,9 @@ build:
 configuration:
 - Debug
 - Release
+test:
+  assemblies:
```

Comment:
Any way to avoid this? Seems like if we added a test assembly and forgot to add it here, we would be none the wiser?